### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by leerob**
A little surprised this went as smoothly as it did... but updated everything to the latest dependencies with minor changes.

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -3,5 +3,6 @@ module.exports = {
     bracketSpacing: false,
     printWidth: 120,
     singleQuote: true,
-    tabWidth: 4
+    tabWidth: 4,
+    trailingComma: 'none'
```
Comment (leerob) [inline .prettierrc.js:7]: Prettier changed the default here.

File: package.json [hunk 2]
```diff
@@ -35,36 +35,36 @@
...
-        "@storybook/addon-docs": "5.3.0-rc.11",
-        "@storybook/addon-info": "5.3.0-rc.11",
-        "@storybook/react": "5.3.0-rc.11",
-        "@storybook/storybook-deployer": "2.8.1",
+        "@babel/cli": "7.11.6",
+        "@babel/core": "7.11.6",
+        "@babel/preset-env": "7.11.5",
+        "@babel/preset-react": "7.10.4",
+        "@storybook/addon-docs": "6.0.21",
+        "@storybook/preset-create-react-app": "3.1.4",
```
Comment (leerob) [inline package.json:43]: This was necessary to add with 6.0.

File: packages/input/src/index.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ const StyledContainer = styled.div`
 
 const StyledLabel = styled.label`
     color: #868686;
-    font-family: 'Fira Sans';
+    font-family: -apple-system, BlinkMacSystemFont;
```
Comment (leerob) [inline packages/input/src/index.js:12]: TBH I'm not sure why I was using a Google Font before. This makes more sense.

Comment (manovotny) [reply packages/input/src/index.js:12]: Do we need something more inclusive and not just Mac focused?

https://github.com/twbs/bootstrap/blob/5512dde203cc3719bac20fb59aa269b99986cfee/scss/_variables.scss#L386

Comment (leerob) [reply packages/input/src/index.js:12]: Not a bad idea. I'll update!

**Comment by leerob** [comment]
I also updated the blog post with these changes 👍

**Review by manovotny — APPROVED** [review]

**Pull request merged**